cypress/integration/acceptance/user-activity.specs.js

```diff
@@ -0,0 +1,256 @@
+'use strict';
+
+const moment = require('moment');
+const testConfig = require('../../../test/acceptance/config');
```

Comment:
FYI, I've deleted test/acceptance and test/smoke in the final ticket, so can you move or remove any references to this config file.